Automattic/jetpack
# VideoPress: Implement upload progress bar

Insert the progress bar into the VideoPress upload experience.

It was mostly based on the current implementation, with some code reduction.

#### Changes proposed in this Pull Request:

* Introduce `UploadProgress` component.
* Use it into `VideoPressUploader` render.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Start `Jetpack`
* Navigate to post edit and insert VideoPress (beta) block.
* Upload a video.
* You should see the progress bar.

#### Demo

https://user-images.githubusercontent.com/1663717/177369767-7f389bd2-9a47-489d-a271-b9052e1dcb63.mp4
